internal/renderer/service_util.go

```diff
@@ -341,6 +343,51 @@ func createLbService4Gateway(c *RenderContext, gw *gwapiv1a2.Gateway) *corev1.Se
 		})
 	}
 
+	// Set health check port
+	regexProtocol := `^service\.beta\.kubernetes\.io\/.*health.*protocol$`
+	regexPort := `^service\.beta\.kubernetes\.io\/.*health.*port$`
+
+	annotationRegexProtocol := regexp.MustCompile(regexProtocol)
+	annotationRegexPort := regexp.MustCompile(regexPort)
+
+	var healthCheckPort int32
+	healthCheckProtocol := "TCP"
+
+	// Find health check port
+	for annotationKey, annotation := range c.gwConf.Spec.LoadBalancerServiceAnnotations {
+		match := annotationRegexPort.FindStringSubmatch(annotationKey)
+		if len(match) > 0 {
+			p, err := strconv.ParseInt(annotation, 10, 32)
+			if err != nil {
+				c.log.V(1).Error(err, "%s annotation value can't be parsed as an int", match[0])
```

Comment:
Let's avoid Error here, this problem is not critical. Also, `log.V(1).Error` does not seem to make much sense, why not using `log.Error`? And finally: if the error message uses a format string then you must use `Errorf`. But let's just make it into a simple `log.V(1).Info` and move on.